chrome/locale/en-US/zotero/zotero.ftl

```diff
@@ -13,6 +13,11 @@ command-or-control =
         [macos] { general-key-command }
         *[other] { general-key-control }
     }
+command-or-control-shift =
+    { PLATFORM() ->
+        [macos] { general-key-command } + { general-key-shift }
```

Comment:
Mac shortcuts are given using hyphens (or en dashes if there's a multi-word key) and no spaces:

https://support.apple.com/en-us/102650

Not sure how it will look without spaces with the input fields, but let's see.

(Will need to adjust `preferences-advanced-keys-modifier` too.)

Reply:
Thanks, fixed.

We now have a `shortcut-separator` in zotero.ftl.

MacOS:

<img width="582" alt="image" src="https://github.com/user-attachments/assets/242aad3f-59ab-4527-acd6-901830f5c318" />

Win:

<img width="595" alt="image" src="https://github.com/user-attachments/assets/6c34b84e-7823-486e-adb8-5619c5937a12" />